ui/models.py

```diff
@@ -219,7 +221,11 @@ def subtitle_key(self, language='en'):
         Returns:
             str: S3 object key
         """
-        return 'subtitles/{}/subtitle_{}.vtt'.format(self.hexkey, language)
+        return 'subtitles/{key}/subtitles_{key}_{dt}_{lang}.vtt'.format(
```

Comment:
Can you pass in `datetime.now(tz=pytz.UTC)` as an argument so that the function is more deterministic?

Reply:
✅